java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2089_java.json

```diff
@@ -1,27 +1,16 @@
 {
   "title": "HTTP referers should not be relied on",
   "type": "VULNERABILITY",
-  "status": "ready",
+  "status": "deprecated",
```

Comment:
Should we create a ticket to document this change (to stay consistent with the ticket for S2653?